dbms/src/TiDB/Schema/SchemaGetter.cpp

```diff
@@ -157,6 +157,10 @@ void SchemaDiff::deserialize(const String & data)
 {
     Poco::JSON::Parser parser;
     Poco::Dynamic::Var result = parser.parse(data);
+    if (result.isEmpty())
+    {
+        LOG_WARNING(&Poco::Logger::get("SchemaDiff"), "The schema diff deserialize failed {}", data);
```

Comment:
Let's throw an Exception here and let the SchemaSyncService retry instead of making tiflash crash.